modules/apps/29-fee/types/msgs.go

```diff
@@ -156,20 +151,30 @@ func (fee IdentifiedPacketFee) Validate() error {
 		return sdkerrors.Wrap(err, "failed to convert RefundAddress into sdk.AccAddress")
 	}
 
+	// enforce relayer is nil
+	if fee.Relayers != nil {
+		return ErrRelayersNotNil
+	}
+
+	// validate Fee
+	if err := fee.Fee.Validate(); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// Validates that each Fee is valid or if all three Fees are empty or zero
+func (fee Fee) Validate() error {
 	// if any of the fee's are invalid return an error
-	if !fee.Fee.AckFee.IsValid() || !fee.Fee.ReceiveFee.IsValid() || !fee.Fee.TimeoutFee.IsValid() {
+	if !fee.AckFee.IsValid() || !fee.ReceiveFee.IsValid() || !fee.TimeoutFee.IsValid() {
 		return sdkerrors.ErrInvalidCoins
```

Comment:
I think we can wrap these errors too like in [#741](https://github.com/cosmos/ibc-go/pull/741#discussion_r786071334)

Reply:
done